api/bin/chainflip-lp-api/Cargo.toml

```diff
@@ -38,6 +38,7 @@ frame-system = { git = 'https://github.com/chainflip-io/polkadot-sdk.git', tag =
 
 # Local
 chainflip-api = { path = "../../lib" }
+state-chain-runtime = { path = "../../../state-chain/runtime" }
```

Comment:
I don't think this is needed, looks like BlockUpdate is re-exported through chainflip-api. (Same goes for cf-primitives, not strictly necessary to import this here).